[2024-11-05T11:02:53.644+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T11:02:53.682+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_test_v1.test_api manual__2024-11-05T11:02:36.798901+00:00 [queued]>
[2024-11-05T11:02:53.698+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_test_v1.test_api manual__2024-11-05T11:02:36.798901+00:00 [queued]>
[2024-11-05T11:02:53.699+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-05T11:02:53.721+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): test_api> on 2024-11-05 11:02:36.798901+00:00
[2024-11-05T11:02:53.739+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_test_v1', 'test_api', 'manual__2024-11-05T11:02:36.798901+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/Weather_etl_dag.py', '--cfg-path', '/tmp/tmp1866lbco']
[2024-11-05T11:02:53.750+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask test_api
[2024-11-05T11:02:53.750+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=225) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-05T11:02:53.753+0000] {standard_task_runner.py:72} INFO - Started process 232 to run task
[2024-11-05T11:02:53.849+0000] {task_command.py:467} INFO - Running <TaskInstance: api_test_v1.test_api manual__2024-11-05T11:02:36.798901+00:00 [running]> on host 68d32cd9cfe4
[2024-11-05T11:02:54.707+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Simo' AIRFLOW_CTX_DAG_ID='api_test_v1' AIRFLOW_CTX_TASK_ID='test_api' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T11:02:36.798901+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-05T11:02:36.798901+00:00'
[2024-11-05T11:02:54.709+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T11:02:54.729+0000] {logging_mixin.py:190} INFO - <method 'date' of 'datetime.datetime' objects>
[2024-11-05T11:02:54.730+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-05T11:02:54.769+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T11:02:54.770+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_test_v1, task_id=test_api, run_id=manual__2024-11-05T11:02:36.798901+00:00, execution_date=20241105T110236, start_date=20241105T110253, end_date=20241105T110254
[2024-11-05T11:02:54.816+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T11:02:54.849+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T11:02:54.852+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
